src/dolphin/workflows/config/_unwrap_options.py

```diff
@@ -0,0 +1,255 @@
+        ),
+    )
+    tile_overlap: tuple[int, int] = Field(
+        (0, 0),
+        description=(
+            "Amount of tile overlap (in pixels) along the (row, col) directions."
+        ),
+    )
+    n_parallel_tiles: int = Field(
+        1,
+        description="Number of tiles to unwrap in parallel for each interferogram.",
+    )
+    init_method: Literal["mcf", "mst"] = Field(
+        "mcf",
+        description="Initialization method for SNAPHU.",
+    )
+    cost: Literal["defo", "smooth"] = Field(
+        "smooth",
+        description="Statistical cost mode method for SNAPHU.",
+    )
+
+
+class TophuOptions(BaseModel, extra="forbid"):
+    ntiles: tuple[int, int] = Field(
+        (1, 1),
+        description="Number of tiles to split the inputs into",
+    )
+    downsample_factor: tuple[int, int] = Field(
+        (1, 1),
+        description="Extra multilook factor to use for the coarse unwrap.",
+    )
+    init_method: Literal["mcf", "mst"] = Field(
+        "mcf",
+        description="Initialization method for SNAPHU.",
+    )
+    cost: Literal["defo", "smooth"] = Field(
+        "smooth",
+        description="Statistical cost mode method for SNAPHU.",
+    )
+
+    _to_tuple = field_validator("ntiles", "downsample_factor", mode="before")(to_tuple)
+
+
+class SpurtGeneralSettings(BaseModel):
+    use_tiles: bool = Field(default=True, description="Tile up data spatially.")
+    intermediate_folder: Path = Field(
+        default=Path("./emcf_tmp"), description="Path to intermediate folder."
+    )
+    output_folder: Path = Field(
+        default=Path("./emcf"), description="Path to output folder."
+    )
+
+
+class SpurtTilerSettings(BaseModel):
+    """Class for holding tile generation settings."""
+
+    max_tiles: int = Field(16, description="Maximum number of tiles allowed.", ge=1)
+    target_points_for_generation: int = Field(
+        120000,
+        description="Number of points used for determining tiles based on density.",
+        gt=0,
+    )
+    target_points_per_tile: int = Field(
+        800000, description="Target points per tile when generating tiles.", gt=0
+    )
+    dilation_factor: float = Field(
+        0.05,
+        description=(
+            "Dilation factor of non-overlapping tiles. "
+            "0.05 would lead to 5 percent dilation of the tile."
+        ),
+        ge=0.0,
+    )
+
+
+class SpurtSolverSettings(BaseModel):
+    worker_count: int = Field(
+        default=0,
+        description=(
+            "Number of workers for temporal unwrapping in parallel. "
+            "Set value to <=0 to let workflow use default workers (ncpus - 1)."
+        ),
+    )
+    links_per_batch: int = Field(
+        default=10000,
+        description=(
+            "Temporal unwrapping operations over spatial links are performed in"
+            " batches and each batch is solved in parallel."
+        ),
```

Comment:
When you write long multi-line strings like this, sometimes you put whitespace at the end each line and sometimes you put it at the beginning of each line.

IMO you should pick one convention and stick with it. Missing whitespace is a lot easier to spot if you adopt a consistent convention.